server/auth/types/proxy/proxy_auth.ts

```diff
@@ -75,7 +73,7 @@ export class ProxyAuthentication extends AuthenticationType {
       : false;
   }
 
-  getAdditionalAuthHeader(request: OpenSearchDashboardsRequest): any {
+  async getAdditionalAuthHeader(request: OpenSearchDashboardsRequest): Promise<any> {
```

Comment:
Why is this change needed?

Reply:
`getAdditionalAuthHeader` is changed to async function. Because multi_auth implementation need to call getCookie mehtod, which is async function.